README.md

```diff
@@ -79,18 +88,17 @@ afterAll(() => unmock.default.off());
 describe("getHoroscope", () => {
   it(
     "augments the API call with seen=false",
-    runner(async () => {
-      zodiac.spy.resetHistory();
-      zodiac.state(withCodes(200));
+    runner(async () => { // 5
+      zodiac.state(withCodes(200)); // 6
       const res = await getHoroscope();
-      expect(res).toMatchObject(JSON.parse(zodiac.spy.getResponseBody()));
+      expect(res).toMatchObject(JSON.parse(zodiac.spy.getResponseBody())); // 7
```

Comment:
What are the numbers?

Reply:
Ah nevermind, just saw [this](https://github.com/unmock/unmock-js/pull/323/files#diff-04c6e90faac2675aa89e2176d2eec7d8R101).

I would either introduce this explanation either or at least explain the numbering comments beforehand because it's a bit confusing.